frontend/src/components/TransferForm.tsx

```diff
@@ -11,41 +11,37 @@ function TransferForm() {
   // TODO: get merchant wallet id after login/accounts are implemented
 
   const merchantWalletId = "1000130251";
+
+  const makeTransfer = async (e: React.SyntheticEvent) => {
+    e.preventDefault();
+    await axiosInstance
+      .post(`/transfer`, {
+        walletId: merchantWalletId,
+        amount: amountToSend,
+        currency: "USD",
+        destinationType: "blockchain",
+        destinationAddress: destinationAddress,
+        chain: "ETH",
+      })
+      .then((response) => {
+        console.log(
+          "Received successful response after transfer",
+          response.data
```

Comment:
this always gets logged to the console, but I'm unable to get `response.data.transactionHash` for example, even when it's available
![image](https://user-images.githubusercontent.com/6494625/124410756-fdeef800-dcff-11eb-9da6-8ae65ba8b573.png)